controllers/userController.js

```diff
@@ -55,13 +64,60 @@ const loginUser = async (req, res) => {
     
     if (dbuser == null) {
         //user does not exist
-        res.send("User does not exists")
+        res.json({
+            message: "User does not exists"
+        })
     } else {
         // user exists
         bcrypt.compare(req.body.password, dbuser.password, async (err, result) => {
             if (err) return err;
             if(result) {
-                res.send("Login Success\n" + "Mongo Id is: " + dbuser._id)
+                const userExists = await tokenModel.findOne({user_id : bodyUsername});
+
+                if(userExists != null) {
```

Comment:
for ignore much if else you can just simply check it like userExists == null return error instead of open an extra block